src/components/editor/AuthorListController.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController'
+
+export default class AuthorListController extends FalcorController {
+  static getFalcorPath(params) {
+    return [];
+  }
+
+  render() {
+    return <div><p>Hi</p></div>;
+  }
+}
```

Comment:
For some reason your text editor is not adding newlines to the end of the file (required by POSIX standard). Can you check this out.